ProductComment.php

```diff
@@ -277,19 +277,44 @@ public static function getGradedCommentNumber($id_product)
      *
      * @return array Comments
      */
-    public static function getByValidate($validate = '0', $deleted = false)
+    public static function getByValidate($validate = '0', $p = null, $limit = null, $skip_validate = false)
```

Comment:
Please don't remove $deleted parameter ;)

Reply:
@Progi1984 

Why? $deleted param is unused, why it should stay ?

Reply:
Because, it's a breaking change to remove it ;)

Reply:
@Progi1984 

$deleted param has been restored

Reply:
:+1: